tutorials/Multi_objective_multi_fidelity_BO.ipynb

```diff
@@ -0,0 +1,1720 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "----------------------------\n",
+      "Iteration Number is  1\n",
+      "----------------------------\n",
+      "Iteration Number is  2\n",
+      "----------------------------\n",
+      "Iteration Number is  3\n",
+      "----------------------------\n",
+      "Iteration Number is  4\n",
+      "----------------------------\n",
+      "Iteration Number is  5\n",
+      "----------------------------\n",
+      "Iteration Number is  6\n",
+      "----------------------------\n",
+      "Iteration Number is  7\n",
+      "----------------------------\n",
+      "Iteration Number is  8\n",
+      "----------------------------\n",
+      "Iteration Number is  9\n",
+      "----------------------------\n",
+      "Iteration Number is  10\n",
+      "----------------------------\n",
+      "Iteration Number is  11\n",
+      "----------------------------\n",
+      "Iteration Number is  12\n",
+      "----------------------------\n",
+      "Iteration Number is  13\n",
+      "----------------------------\n",
+      "Iteration Number is  14\n",
+      "----------------------------\n",
+      "Iteration Number is  15\n",
+      "----------------------------\n",
+      "Iteration Number is  16\n",
+      "----------------------------\n",
+      "Iteration Number is  17\n",
+      "----------------------------\n",
+      "Iteration Number is  18\n",
+      "----------------------------\n",
+      "Iteration Number is  19\n",
+      "----------------------------\n",
+      "Iteration Number is  20\n",
+      "----------------------------\n",
+      "Iteration Number is  21\n",
+      "----------------------------\n",
+      "Iteration Number is  22\n",
+      "----------------------------\n",
+      "Iteration Number is  23\n",
+      "----------------------------\n",
+      "Iteration Number is  24\n",
+      "----------------------------\n",
+      "Iteration Number is  25\n",
+      "----------------------------\n",
+      "Iteration Number is  26\n",
+      "----------------------------\n",
+      "Iteration Number is  27\n",
+      "----------------------------\n",
+      "Iteration Number is  28\n",
+      "----------------------------\n",
+      "Iteration Number is  29\n",
+      "----------------------------\n",
+      "Iteration Number is  30\n"
+     ]
+    }
+   ],
+   "source": [
+    "# MOMF Opt run\n",
+    "from botorch import fit_gpytorch_model\n",
+    "from botorch.sampling.samplers import SobolQMCNormalSampler\n",
+    "\n",
+    "verbose = True\n",
+    "train_x = torch.zeros(n_INIT + n_BATCH, dim_x).to(**tkwargs)  # Intializing train_x to zero\n",
+    "train_obj = torch.zeros(n_INIT + n_BATCH, dim_y).to(**tkwargs)  # Intializing train_obj to zero\n",
```

Comment:
This cell (and much of the experiment setup below) will only work for BATCH_SIZE=1. I would recommend generalizing this so that users can simply modify the batch size and still use the code